### ISSUE TITLE
conflicting action statements

### ISSUE THREAD

**Issue by damianoneill** [suzuki-shunsuke/ansible-gvm#28]
Hi @suzuki-shunsuke I am getting the following error, using ansible version 2.10.4

```console
ERROR! conflicting action statements: file, items

The error appears to be in '/Users/doneill/.ansible/roles/suzuki-shunsuke.gvm/tasks/main.yml': line 34, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

# https://github.com/moovweb/gvm/blob/master/scripts/gvm-default
- name: create directories (gvm-default)
  ^ here
```

**Comment by damianoneill** [comment]
Hi @suzuki-shunsuke checking in to see if you have had a chance to look at this?

**Comment by suzuki-shunsuke** [comment]
Hi, sorry for late reply.
I'll check.

**Comment by suzuki-shunsuke** [comment]
https://galaxy.ansible.com/suzuki-shunsuke/gvm

I have fixed the problem and released v6.0.0
The problem was due to the breaking change of Ansible.
Please try to use v6.0.0.

**Comment by damianoneill** [comment]
Thanks @suzuki-shunsuke this resolved the problem.
